src/c_kzg_4844.h

```diff
@@ -254,6 +254,10 @@ C_KZG_RET verify_sample_proof(
     const KZGSettings *s
 );
 
+C_KZG_RET get_2d_samples(
+    Bytes32 *data, const Blob *blobs, const KZGSettings *s
+);
+
```

Comment:
Yeah sorry about that. I forgot to expose those functions in the header. The latest version of the `das` branch should have those now.